core/reloader/reloader-release.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: release-reloader
+  namespace: flux-config
+spec:
+  chart:
+    spec:
+      chart: reloader
+      version: 1.0.72
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: repository-reloader
+  install:
+    createNamespace: true
+  interval: 5m0s
+  releaseName: reloader-service
+  targetNamespace: reloader
+  values:
+    reloader:
+      deployment:
+        enableHA: true
+        replicas: 2
+        tolerations:
+          - key: "CriticalAddonsOnly"
+            operator: "Equal"
+            value: "true"
+            effect: "NoSchedule"
+        affinity:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+              - matchExpressions:
+                - key: kubernetes.azure.com/mode
+                  operator: In
+                  values:
+                    - 'system'
+        nodeSelector:
+          kubernetes.io/os: "linux"
+      podDisruptionBudget:
+        enabled: false
```

Comment:
PDP should be enabled right? with min replicas above at 3 I think we autoscaling to 6?

min available at all times can be 1 I think thats fine?

Reply:
Yes, it should be enabled... Corrected now.